src/include/fmgr.h

```diff
@@ -18,6 +18,7 @@
 #ifndef FMGR_H
 #define FMGR_H
 
+#include "nodes/pg_list.h"
```

Comment:
Why do we need this?

Reply:
`fmgr.h` is a header file used by `execSRF.c`. And `List` hasn't been used in `execSRF.c` until we added our pivot logic. So we need to include this to `fmgr.h`